tests/test_job_management.py

```diff
@@ -1395,7 +1395,7 @@ def test_email_scws_list(dispatcher_long_living_fixture,
         'please check the inputs')
     error_message_scw_list_missing_file = (
         'scw_list file was expected to be passed, but it has not been found, '
-        'please check the inputs')
+        'please check the inputs, content of the temporary directory is []')
```

Comment:
I do not suppose we content of temp directory in this case, when error is something else.

Reply:
You mean we're not interested? You're right, but the exception happens exactly when the inputs are parsed, and the content of the temp_dir is reported.

Reply:
> You mean we're not interested in it? You're right, but the exception happens exactly when the inputs are parsed, and the content of the temp_dir is reported.

there is no need to report content of `temp_dir` unless the problem was in deleting the `temp_dir`. We could also separately catch just that exact exception.

Reply:
Ok, didn't consider that was referred to a case with erroneous inputs